app-text/poppler/poppler-0.50.0.recipe

```diff
@@ -0,0 +1,298 @@
+SUMMARY="A PDF rendering library"
+DESCRIPTION="Poppler is a PDF rendering library based on the xpdf-3.0 code \
+base."
+HOMEPAGE="https://poppler.freedesktop.org/"
+COPYRIGHT="2002-2008 Albert Astals Cid
+	2005 Kristian Høgsberg
+	2001-2002 Derek Noonburg (xpdf)"
+LICENSE="GNU GPL v2"
+REVISION="1"
+DATAVERSION=0.4.7
+SOURCE_URI="https://poppler.freedesktop.org/poppler-$portVersion.tar.xz"
+SOURCE_URI_2="https://poppler.freedesktop.org/poppler-data-$DATAVERSION.tar.gz"
```

Comment:
I would recommend moving the poppler_data sub-package to a separate recipe.
We would have:
* app-text/poppler/poppler-0.50.0.recipe
* app-text/poppler-data/poppler_data-0.4.7.recipe

BTW, Gentoo also has a separate poppler-data package.